core/order_test.go

```diff
@@ -1,6 +1,8 @@
 package core_test
 
 import (
+	"github.com/OpenBazaar/openbazaar-go/test/factory"
+	"github.com/go-errors/errors"
```

Comment:
Organize imports like so:
```
import (
  // all 3rd party library imports

  // all stdlib imports
)
```
(with the newline separating the groups)

Reply:
done